presto-native-execution/presto_cpp/main/http/HttpServer.cpp

```diff
@@ -141,14 +141,13 @@ proxygen::HTTPServer::IPConfig HttpsConfig::ipConfig() const {
 HttpServer::HttpServer(
     std::unique_ptr<HttpConfig> httpConfig,
     std::unique_ptr<HttpsConfig> httpsConfig,
-    int httpExecThreads)
+    int httpExecThreads,
+    const std::shared_ptr<folly::IOThreadPoolExecutor>& ioThreadPool)
     : httpConfig_(std::move(httpConfig)),
       httpsConfig_(std::move(httpsConfig)),
       httpExecThreads_(httpExecThreads),
```

Comment:
httpExecThreads_ is a const?

Reply:
No, but the sp is passed in as const ref